config/locales/event_types/en.yml

```diff
@@ -13,6 +13,7 @@ en:
     email_deleted: Email address deleted
     new_personal_key: Personal key changed
     password_changed: Password changed
+    password_invalidated: Password invalidated by %{app_name}
```

Comment:
Couple thoughts:
1. password reset over password invalidated.
2. while the app sent a security event to login, it was login that took the action to invalidate the password. So either:
"Security event reported by #app name#, password has been reset" 
or
Just leave it as "Password reset"

Reply:
The "events" model doesn't have a the columns to let us tie back to an SP, so I'll just make it "password reset"

Reply:
Oh. I mistook the %{app_name} in the code to be the app name of the SP.  If that's always going to be "login.gov" to the user i'm fine with that.  i.e. Password reset by login.gov.

Reply:
Ok I'll bring login.gov back :)

Reply:
Screenshot with newest copy

<img width="826" alt="Screen Shot 2020-09-03 at 3 45 45 PM" src="https://user-images.githubusercontent.com/458784/92180727-9d034480-edfc-11ea-9420-f4431cf66e76.png">